src/System.Buffers/src/Configurations.props

```diff
@@ -4,15 +4,13 @@
     <PackageConfigurations>
       netstandard1.1;
       netstandard;
-      netcoreapp2.0-Windows_NT;
-      netcoreapp2.0-Unix;
-      uap10.0.16299-Windows_NT;
-      uap10.0.16299aot-Windows_NT;
     </PackageConfigurations>
     <BuildConfigurations>
       $(PackageConfigurations);
       netcoreapp-Windows_NT;
       netcoreapp-Unix;
+      uap10.0.16299-Windows_NT;
```

Comment:
Same here, you should just remove these 2 since we will only have placeholders now.